app/views/user_mailer/invited_to_collection_notification.html.erb

```diff
@@ -1,15 +1,12 @@
 <% content_for :message do %>
-  <p>Dear <%= style_bold(@user.default_pseud.byline) %>,</p>
+  <p><%= t(".greeting", name: style_bold(@user.default_pseud.byline)).html_safe %></p>
 
-  <p>The collection maintainers of <%= style_link(@collection.title, collection_url(@collection)) %> would
-    like to include your work (<%= style_link(@work.title.html_safe, work_url(@work)) %>) in their collection!</p>
+  <p><%= t(".part1", collection: style_link(@collection.title, collection_url(@collection)).html_safe, work: style_link(@work.title.html_safe, work_url(@work)).html_safe).html_safe %></p>
 
-  <p>To approve or reject this request, please visit your <%= style_link("Collection Items page", user_collection_items_url(@user)) %>.</p>
+  <p><%= t(".html.part2", link: style_link(t(".html.link_text").html_safe, user_collection_items_url(@user)).html_safe).html_safe %></p>
```

Comment:
There are multiple links in this email, so we should probably be more specific with our names. `collection_items` in place of `link` and `collection_items_text` in place of `link_text` might be consistent with the approach in #3889
 
(If we go with that, changing line 8 to use `style_link(t(".html.preferences_text")` would keep things consistent within the file.)

Reply:
Fixed